scripts/arm32_ci_script.sh

```diff
@@ -146,9 +146,8 @@ function cross_build_corefx {
     git am < "$__ARMRootfsMountPath"/dotnet/setenv/corefx_cross.patch
 
     #Cross building for emulator rootfs
-    ROOTFS_DIR="$__ARMRootfsMountPath" CPLUS_INCLUDE_PATH=$LINUX_ARM_INCPATH CXXFLAGS=$LINUX_ARM_CXXFLAGS
-    BUILD_NATIVE="$ROOTFS_DIR" ./build-native.sh -buildArch=$__buildArch -$__buildConfig -- cross $__verboseFlag
-    BUILD_MANAGED="$ROOTFS_DIR" ./build-managed.sh -buildArch=$__buildArch -$__buildConfig -skipTests
+    ROOTFS_DIR="$__ARMRootfsMountPath" CPLUS_INCLUDE_PATH=$LINUX_ARM_INCPATH CXXFLAGS=$LINUX_ARM_CXXFLAGS ./build-native.sh -buildArch=$__buildArch -$__buildConfig -- cross $__verboseFlag
+    ROOTFS_DIR="$__ARMRootfsMountPath" CPLUS_INCLUDE_PATH=$LINUX_ARM_INCPATH CXXFLAGS=$LINUX_ARM_CXXFLAGS ./build-managed.sh -buildArch=$__buildArch -$__buildConfig -skipTests
```

Comment:
I think you wrote below codes to run managed build
`... ./build-managed.sh -buildArch=$__buildArch -$__buildConfig -skipTests`
But I cannot find to process '-buildArch, -$__buildConfig and -skipTests' in 'build-managed.sh'

build-managed.sh
```
while :; do
    if [ $# -le 0 ]; then
        break
    fi

    lowerI="$(echo $1 | awk '{print tolower($0)}')"
    case $lowerI in
        freebsd)
            __TestNugetRuntimeId="$__TestNugetRuntimeId=osx.10.10-x64"
            __BuildOS="$__BuildOS=freebsd"
            __TargetOS="$__TargetOS=freebsd"
            ;;
        netbsd)
            __TestNugetRuntimeId="$__TestNugetRuntimeId=osx.10.10-x64"
            __BuildOS="$__BuildOS=netbsd"
            __TargetOS="$__TargetOS=netbsd"
            ;;
        osx)
            __TestNugetRuntimeId="$__TestNugetRuntimeId=osx.10.10-x64"
            __BuildOS="$__BuildOS=osx"
            __TargetOS="$__TargetOS=osx"
            ;;
        windows)
            __TestNugetRuntimeId="$__TestNugetRuntimeId=win7-x64"
            __BuildOS="$__BuildOS=windows_nt"
            __TargetOS="$__TargetOS=Windows_NT"
            ;;
        useservergc)
            __ServerGC=1
            ;;
        *)
            __UnprocessedBuildArgs="$__UnprocessedBuildArgs $1"
    esac

    shift
done
```